tests/integration/update-for-workloads.yaml

```diff
@@ -1,5 +1,4 @@
 deployment:
-  imagePullPolicy: Never
```

Comment:
It looks like this shouldn't break local integration tests, but there may also have been a reason why we set it to `Never` instead of default. Have you tried running local integration tests with this setup?

Reply:
nope, only CI